README.md

```diff
@@ -9,16 +9,14 @@
 # 📗 Table of Contents
 
 - [📗 Table of Contents](#-table-of-contents)
-- [📖 \[Vet_clinic\] ](#-vet_clinic-)
+- [📖 \[School_library\] ](#-School_library-)
   - [🛠 Built With ](#-built-with-)
     - [Tech Stack ](#tech-stack-)
     - [Key Features ](#key-features-)
   - [💻 Getting Started ](#-getting-started-)
     - [Prerequisites](#prerequisites)
     - [Setup](#setup)
     - [Install](#install)
-    - [Usage](#usage)
-    - [Run test](#run-test)
   - [👥 Authors ](#-authors-)
```

Comment:
- [ ] [**OPTIONAL**] Good job documenting your project 👏 , but your README documentation should be more professional, Kindly ensure that the `Authors` section is found in your README, it will make it even more professional 😄 
- [ ] See the example below

![Capture](https://github.com/Otegaa/school_library/assets/80458199/a8888d5d-f920-4ca7-862c-9dfd5ced424b)